src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Figure circle = new Circle();
+        Figure rectangle = new Rectangle();
+        Figure rightTriangle = new RightTriangle();
+        Figure square = new Square();
+        Figure isoscelesTrapezoid = new Figure();
```

Comment:
The instantiation of `Figure` here is incorrect because `Figure` is likely an abstract class or interface and should not be instantiated directly. Instead, create an instance of a concrete subclass.